assets/js/src/site/plugins/pum.js

```diff
@@ -11,6 +11,7 @@ var PUM;
         default_theme: '0',
         home_url: '/',
         version: 1.7,
+		pm_dir_url: '',
```

Comment:
This remnants of your code formatting issues?

Reply:
This is what I was referring to over in Slack. My line follows our tab requirements. Two tabs. The lines around it are using 8 spaces.